studio/components/interfaces/BillingV2/Usage/Infrastructure.tsx

```diff
@@ -185,8 +184,8 @@ const Infrastructure = ({ projectRef }: InfrastructureProps) => {
               )}
               <div className="space-y-1">
                 <div className="flex flex-row justify-between">
-                  {attribute.key === 'disk_io_budget' ? (
-                    <p>IO Budget remaining</p>
+                  {attribute.key === 'disk_io_consumption' ? (
+                    <p>IO Budget consumed</p>
```

Comment:
Could we simplify this to "IO Consumed"? Throwing in the term "budget" seems unnecessary now.